cli/generate.go

```diff
@@ -73,13 +73,17 @@ var generateCmd = cli.Command{
 			Name:  "exclude,x",
 			Usage: "Files or directories to exclude from this run, relative to the chart.",
 		},
+		cli.BoolFlag{
+			Name:  "force,f",
+			Usage: "Forces to run generate subcommands",
```

Comment:
force for generate command makes sense when the template command is present in the generate header , will change accordingly